profile/README.md

```diff
@@ -1,12 +1,39 @@
-## Hi there 👋
+## Welcome to the Autoware Foundation! 👋
+![Autoware stack](https://static.wixstatic.com/media/984e93_552e338be28543c7949717053cc3f11f~mv2.png/v1/crop/x_0,y_1,w_1500,h_879/fill/w_1079,h_633,al_c,usm_0.66_1.00_0.01,enc_auto/Autoware-GFX_edited.png)
```

Comment:
```suggestion
![Autoware stack](https://static.wixstatic.com/media/984e93_552e338be28543c7949717053cc3f11f~mv2.png/v1/crop/x_0,y_1,w_1500,h_879/fill/w_1079,h_633,al_c,usm_0.66_1.00_0.01,enc_auto/Autoware-GFX_edited.png)
```
Wow this image needs to be updated, it looks so outdated in style. I will create and send one today.

Reply:
I put that in more as a placeholder, and was thinking to replace it with a proper AWF banner image. 

I had thought it would be better to add an Autoware stack diagram to the documentation and perhaps the Core/Universe repositories, but I leave that decision with you guys (@BonoloAWF - Any thoughts on this?)